docs/json/radarr/cf/av1.json

```diff
@@ -1,5 +1,8 @@
 {
   "trash_id": "cae4ca30163749b891686f95532519bd",
+  "trash_scores": {
```

Comment:
missing default and legacy

Reply:
because they don't have a default/legacy score and the sync apps treat a missing `trash_score` as 0.
we are not "allowed" to use a value of 0 for `trash_score`